src/components/V4HomePage.js

```diff
@@ -125,10 +126,12 @@ const HomepageWrapper = ({
           <Mission missionData={missionData} causeId={causeId} />
           <SecuritySection securityData={Security} />
           <FinancialsComponent financialsData={Financials} />
-          <EndorsementsComponent
-            endorsementsData={Endorsements}
-            causeId={causeId}
-          />
+          {Endorsements ? (
```

Comment:
I don't think this works. In this case, we want to optionally hide only the big quote (not all endorsements), so maybe key on Endorsments.quote? Please add a simple test, too. Feel free to break out into a separate PR if you want.

Reply:
I didn't understand what part to make optional 😂, will adjust.